Config/HoloLens/HoloLensEngine.ini

```diff
@@ -0,0 +1,31 @@
+
+
+[/Script/HoloLensPlatformEditor.HoloLensTargetSettings]
```

Comment:
Random but what is this used for?

Reply:
It is generated every time you run the project. I just decided to add it so I wouldn't delete it again